src/Database/QueryCompiler.php

```diff
@@ -51,7 +52,7 @@ class QueryCompiler
      * @var array<string>
      */
     protected array $_selectParts = [
-        'with', 'select', 'from', 'join', 'where', 'group', 'having', 'window', 'order',
+        'comment', 'with', 'select', 'from', 'join', 'where', 'group', 'having', 'window', 'order',
```

Comment:
Shouldn't this order match the order in the query parts array?

Reply:
Right, I will adjust the order in the query parts as well to match this.